doc/how_to_contribute/how_to_contribute_to_site.rst

```diff
@@ -25,59 +26,61 @@ This should make it easy for you to develop changes to this site and make sure y
 
 1. Build and view the website locally
 
-To build the website locally run this command:
+  To build the website locally run this command:
 
-.. code-block:: bash
+  .. code-block:: bash
 
-  make html
-  # Or run the following line if you want to automatically rebuild the website on new changes
-  while inotifywait -re modify,move,create,delete .; do
     make html
-  done
+    # Or run the following lines if you want to automatically rebuild the website on new changes
+    while inotifywait -re modify,move,create,delete .; do
+      make html
+    done
 
-Then you can open the local build of the site in your web browser from: ``./build/html/index.html``. For example, to open the local site in Firefox the command would be:
+  Then you can open the local build of the site in your web browser from: ``./build/html/index.html``. For example, to open the local site in Firefox the command would be:
 
-.. code-block:: bash
+  .. code-block:: bash
 
-  firefox ./build/html/index.html
+    xdg-open ./build/html/index.html
```

Comment:
The sentence says "to open the local site in Firefox"? The user would need to install xdg-utils (I did in the container I was working in). Maybe just change Firefox to say "your preferred browser"?